src/app/functions/server/onerep.ts

```diff
@@ -118,6 +118,16 @@ async function onerepFetch(
   if (!onerepApiKey) {
     throw new Error("ONEREP_API_KEY env var not set");
   }
+
+  //If mock, remove the first slash so that it doesn't overwrite the path
```

Comment:
I think this is OK, but it can be easier and safer to deal use the [URL interface](https://developer.mozilla.org/en-US/docs/Web/API/URL) (which is standard in both browser and Node JS environments) instead of manipulating URLs as strings.

Reply:
I tried to use it, but it requires that the path does not have a leading '/', o/w it overwrites the path in the ONEREP_API_BASE (so, locally it would overwrite to become localhost:port/path, instead of localhost:port/otherPath/path). But the real endpoint won't work w/o leading '/'. This is why I opted for a conditional removal of the slash.